recipes/libgfortran/meta.yaml

```diff
@@ -1,61 +0,0 @@
-{% set libgfortran_version = [3, 0, 0] %}
```

Comment:
Unlucky PR! Can you rebase this on top of master to remove this incorrect history?